schemas/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment/jsonschema/1-0-0

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for IAB spiders&robots enrichment",
```

Comment:
@rzats could you please normalize this one as well (capitalize, white spaces)

Reply:
fixed!